lib/themes/dosomething/paraneue_dosomething/js/components/ReactionsList.js

```diff
@@ -0,0 +1,158 @@
+const React = require('react');
+
+import Reaction from './Reaction';
+import ApiClient from '../utilities/ApiClient';
+import Setting from '../utilities/Setting';
```

Comment:
Since this is a function, let's keep it lowercase: `import setting from '../utilities/Setting'`. And maybe we should move it into that `helpers.js` file we made to reduce confusion with the capitalized filename.